internal/cli/datafederation/logs.go

```diff
@@ -67,7 +68,12 @@ func (opts *LogOpts) Run() error {
 
 // atlas dataFederation log [--projectId projectId].
 func LogBuilder() *cobra.Command {
-	opts := &LogOpts{}
+	opts := &LogOpts{
+		DownloaderOpts: cli.DownloaderOpts{
+			Fs:  afero.NewOsFs(),
+			Out: "-", // stdout
```

Comment:
do we need this? isn't the default stout? I am asking because if the answer is no, we should update also the other commands that download logs

Reply:
I've copied this from `internal/cli/deployments/logs.go`.

It doesn't look like it's the default, the default is ` `, not `-`.
I can create a followup PR to fix it everywhere

Reply:
https://jira.mongodb.org/browse/CLOUDP-275665